.github/pull_request_template.md

```diff
@@ -1,36 +1,33 @@
-### Pull Request
-
-#### Description
-> Please provide a summary of your changes here. 
+> Please provide a summary of your changes here.
 > * Which issue is this pull request belonging to? (*Refer to issue here*)
 > * How is it solving the issue?
-> * Did you add any new dependencies that are required for your change?
-
-#### Commit Messages  
->    The commit messages should follow the syntax for [auto-closing keywords](https://help.github.com/en/articles/closing-issues-using-keywords). 
->    It is preferred if you reference the issue in your commit message.
+> * Did you add or update any new dependencies that are required for your change?
 
-#### Request Reviewer
-> You can add desired reviewers here with an @mention.  
+### Request Reviewer
+> You can add desired reviewers here with an @mention.
 > For a full list of reviewers check out the *Committers* slot on [the project site](https://projects.eclipse.org/projects/technology.sw360.antenna/who).
```

Comment:
How to find out, what the github-ids for the committers is? Maybe it would be good to have a list in the wiki for that.